backend/app/routes.py

```diff
@@ -248,6 +252,14 @@ def aggregate_travel_times(start_node, end_node, start_time, end_time, start_dat
         FROM corridor_period_daily_avg_tt 
     '''
 
+    if is_holiday == 1:

```

Comment:
What is this param expecting? Should the value be `1`? or `true`? Be explicit here. Let's go with `true` or `false`, case insensitive.